pkg/packagekit/updates.jsx

```diff
@@ -1472,23 +1473,23 @@ class OsUpdates extends React.Component {
             });
 
             return (
-                <>
+                <Stack hasGutter>
```

Comment:
There's too much vertical space between items. I think the `hasGutter` is adding too much. It's either that, and/or one of the other elements adding vertical space between items as well.

Reply:
Removed `hasGutter`. Looks better now